.github/actions/prepare-metal-run/action.yml

```diff
@@ -9,6 +9,10 @@ inputs:
     description: "Whether to load with profiler"
     required: false
     default: 'false'
+  python-version:
+    description: 'Which version of Python to use to run the tests.'
+    required: false
+    default: '3.8'
```

Comment:
Makes sense to have this as an argument, but I believe we can generally simplify our compatibility strategy by tying ourselves to the default `python3` version for a given Ubuntu distro.

- 3.8 on 20.04
- 3.10 on 22.04
- 3.12 on 24.04

Just a note for the future. For example, once we get `3.10` working, there wouldn't really be a need to explicitly continue support `3.8` on `22.04`.